Source/Extensions/Blazorise.Charts.Streaming/wwwroot/charts.streaming.js

```diff
@@ -96,4 +94,47 @@ function getStreamingOptions(dotNetAdapter, vertical, chartOptions, streamOption
     }
 
     return horizontalScalesOptions;
+}
+
+function getRealtimeOptions(chart) {
+    if (chart && chart.options && chart.options.scales) {
+        const scales = chart.options.scales;
+
+        if (scales.x && scales.x.realtime) {
+            return scales.x.realtime;
+        }
+        else if (scales.x && scales.x.realtime) {
+            return scales.x.realtime;
+        }
+    }
+
+    return null;
+}
+
+export function pause(canvasId, animate) {
+    const chart = getChart(canvasId);
+
+    if (chart) {
+        const realtimeOptions = getRealtimeOptions(chart);
+
+        if (realtimeOptions) {
+            realtimeOptions.pause = true;
+
+            chart.update(animate ? null : "none");
+        }
+    }
+}
+
+export function play(canvasId, animate) {
+    const chart = getChart(canvasId);
+
+    if (chart) {
+        const realtimeOptions = getRealtimeOptions(chart);
+
+        if (realtimeOptions) {
+            realtimeOptions.pause = false;
+
+            chart.update(animate ? null : "none");
```

Comment:
I don't really get why null vs "none" depending on animate... but I imagine this is correct.

Reply:
"none" is a reserver keyword from chartjs. 🤷‍♂️